docs/reference/migration/migrate_8_18.asciidoc

```
@@ -81,6 +81,20 @@ Before this change, in case of shard failures, EQL queries always returned an er
 EQL queries that would previously fail due to shard failures, will now succeed and return partial results. The previous defaults can be restored by setting `xpack.eql.default_allow_partial_results` cluster setting to `false` or setting with `allow_partial_search_results` to `false` in the query request.
 ====
 
+[discrete]
+[[breaking_818_esql_changes]]
+==== ES|QL changes
+
+[[limit_scope_skip_unavailable_setting_remote_clusters]]
+.Limit the scope of the skip_unavailable setting for remote clusters
+[%collapsible]
+====
+*Details* +
+Before this change, ES|QL would honor skip_unavailable setting for nonmatching indices errors at planning time.
+
+*Impact* +
+Remote clusters, regardless of their skip_unavailable setting, will now use the same logic as the local cluster for index expression analysis at plan time
```

We might want to be more explicit and state that rule: any missing concrete index in your index expression will return an error, rather than returning partial results.